examples/rrt_panda.py

```diff
@@ -43,11 +43,11 @@
     options = RRTPlannerOptions(
         max_step_size=0.05,
         max_connection_dist=0.5,
-        rrt_connect=False,
-        bidirectional_rrt=False,
+        rrt_connect=True,
```

Comment:
I would like to keep the defaults as they are in this file. The intention is for people to start with vanilla RRT, see its issues, and explore improvements themselves.